octave 5.1 landed...

Our current swig version produces some issues (spotted by compiling plplot). I can give more details if needed.
Thanks.

```
/rpmbuild/BUILD/plplot-5.14.0/build/bindings/octave/plplot_octaveOCTAVE_wrap.cxx
/home/iurt/rpmbuild/BUILD/plplot-5.14.0/build/bindings/octave/plplot_octaveOCTAVE_wrap.cxx: In function 'octave_value SWIG_Error(int, const char*)':
/home/iurt/rpmbuild/BUILD/plplot-5.14.0/build/bindings/octave/plplot_octaveOCTAVE_wrap.cxx:921:18: error: format not a string literal and no format arguments [-Werror=format-security]
   error(r.c_str());
                  ^
/home/iurt/rpmbuild/BUILD/plplot-5.14.0/build/bindings/octave/plplot_octaveOCTAVE_wrap.cxx: In member function 'virtual octave_value octave_swig_type::map(octave_base_value::unary_mapper_t) const':
/home/iurt/rpmbuild/BUILD/plplot-5.14.0/build/bindings/octave/plplot_octaveOCTAVE_wrap.cxx:1710:66: error: format not a string literal and no format arguments [-Werror=format-security]
         error((opname + std::string(" method not found")).c_str());
```

--- a/Lib/octave/octrun.swg
+++ b/Lib/octave/octrun.swg
@@ -51,7 +51,7 @@ SWIGRUNTIME octave_value SWIG_Error(int code, const char *msg) {
   octave_value type(SWIG_ErrorType(code));
   std::string r = msg;
   r += " (" + type.string_value() + ")";
-  error(r.c_str());
+  error("%s", r.c_str());
   return octave_value(r);
 }
 
@@ -840,7 +840,7 @@ SWIGRUNTIME void swig_acquire_ownership_obj(void *vptr, int own);
       const std::string opname = std::string("__") + octave_base_value::get_umap_name(umap) + std::string("__");
       octave_value ret;
       if (!dispatch_unary_op(opname, ret)) {
-        error((opname + std::string(" method not found")).c_str());
+        error("%s", (opname + std::string(" method not found")).c_str());
         return octave_value();
       }
       return ret;

